src/Jh/DataImportMagento/Writer/ShipmentWriter.php

```diff
@@ -64,6 +70,17 @@ public function writeItem(array $item)
                 ->addObject($shipment->getOrder())
                 ->save();
 
+            if (isset($item['tracking_number'], $item['carrier'])) {
```

Comment:
I was told not to because for this project they will never send multiple tracks.